packages/zend-form/library/Zend/Form/Element/Captcha.php

```diff
@@ -228,7 +228,7 @@ public function getPluginLoader($type)
      */
     public function addPrefixPath($prefix, $path, $type = null)
     {
-        $type = strtoupper($type);
+        $type = is_string($type) ? strtoupper($type) : $type;
```

Comment:
idem

Reply:
oh, for `addPrefixPath` we have to allow passing `null`, so it's not relevant anymore